cmd/oras/internal/display/metadata/interface.go

```diff
@@ -30,3 +31,9 @@ type PushHandler interface {
 type AttachHandler interface {
 	OnCompleted(opts *option.Target, root, subject ocispec.Descriptor) error
 }
+
+// PullHandler handles metadata output for attach events.
+type PullHandler interface {
+	// OnCompleted is called when the pull cmd execution is completed.
+	OnCompleted(opts *option.Target, desc ocispec.Descriptor, layerSkipped bool, _ []model.File) error
+}
```

Comment:
Should the `OnCompleted` method be split to `OnCompleted` and `OnFilePulled`?

Reply:
Why we need to split the output?

Reply:
It is to stream the metadata of the file pulling.

Reply:
Refactored.